src/comp/LeftNav.js

```diff
@@ -0,0 +1,70 @@
+import LeftNavHeader from "./LeftNavHeader.js"
+import LeftNavList from "./LeftNavList.js"
+
+export default function LeftNav({ $target , initState, userName, pageLoadToId, addNavSubDom, deleteNavDom }){
+
+  const $LeftNav = document.createElement('div')
+  $LeftNav.className = 'LeftNav'
+  $target.appendChild($LeftNav)
+  
+  this.state = initState
+
+  this.setState = nextState => {
+    this.state = nextState
+    this.render()
+  }
+
+  this.render = ()=> {
+    $LeftNav.innerHTML = ''
+    new LeftNavHeader({$target : $LeftNav, userName})
+    new LeftNavList({$target : $LeftNav, initState : this.state })
+  }
+
+  this.render()
+
+  // 네브 포커스요소
+  this.NavFocusBox = undefined
```

Comment:
p4: undefined를 할당하신 이유가 있으실까요? 😄 
구글에 검색 해봤는데 이렇게 나와있습니다. 🔍
undefined 는 JS 엔진이 변수에 아무것도 할당되지 않았을 경우 자동적으로 할당하는 값으로, 개발자가 직접 입력해 넣어버리면 본래의 취지와 맞지 않는다. null 은 값이 없음을 나타내는 의도적인 변수로, 개발자가 특정 변수에 값이 없다는 것을 명시할 때 사용한다.

Reply:
p4: undefined로 하고 싶으면 이렇게만 선언해줘도 돼요. 근데 null을 할당해주는게 더 명시적인것 같긴하네요
```suggestion
  this.NavFocusBox;
```